assets/js/components/DashboardEntryPoint.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * DashboardEntryPoint tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { render } from '../../../tests/js/test-utils';
+import { mockCreateComponent } from '../../../tests/js/mock-component-utils';
+import DashboardEntryPoint from './DashboardEntryPoint';
+
+jest.mock( './setup/ModuleSetup', () => mockCreateComponent( 'ModuleSetup' ) );
+jest.mock( './dashboard/DashboardApp', () =>
+	mockCreateComponent( 'DashboardApp' )
+);
+jest.mock( './DashboardMainApp', () =>
+	mockCreateComponent( 'DashboardMainApp' )
+);
+jest.mock( './legacy-notifications/notification-counter', () =>
+	mockCreateComponent( 'NotificationCounter' )
+);
+
+describe( 'DashboardEntryPoint', () => {
+	describe( 'when unified dashboard is enabled', () => {
+		const renderOptions = { features: [ 'unifiedDashboard' ] };
+
+		it( 'should render the unified dashboard', () => {
```

Comment:
This tests organisation seems a little bit over-engineered. Let's better streamline it to use just one describe block with multiple tests.

```js
describe( 'DashboardEntryPoint', () => {
    it( 'should render the unified dashboard when unified dashboard is enabled', () => {
         ...
    } );

    ...
} );
```

Reply:
The same applies to other test files.

Reply:
Thanks, I've updated the tests :+1: